modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/FieldBase.scss

```diff
@@ -9,3 +9,9 @@
 		font-size: 0.875rem;
 	}
 }
+.lfr-ddm-legend {
```

Comment:
@DiegoDOC, can you please add a line break before the class `lfr-ddm-legend`? Usually, this is how we separate CSS classes to make code easy to read. :wink: 

Additionally, can you please send a new pull with just a single commit?

Thanks.